filament/src/fg/fg/PassNode.cpp

```diff
@@ -114,7 +114,7 @@ FrameGraphHandle PassNode::write(FrameGraph& fg, const FrameGraphHandle& handle)
     // record the write
     auto& newNode = fg.getResourceNodeUnchecked(r);
     assert(!newNode.writerIndex.isValid());
-    newNode.writerIndex = r; // needed by move resources
+    newNode.writerIndex = FrameGraphHandle(this->id); // needed by move resources
```

Comment:
Should it still be called index?

Reply:
yes, it's very much the index of the PassNode in the array. I'm using a FrameGraphHandle just as a convenience because it's basically an index with a non-zero default state.